### PR TITLE
First pass at including Integrations (groups) on landing page

### PR DESCRIPTION
**Pull request by joepavitt**
Screenshot available in: https://github.com/flowforge/website/issues/130

### CODE REVIEW

**Comment by sammachin** [comment]
Comment in #130

File: src/index.njk [hunk 1]
```diff
@@ -24,27 +24,40 @@ layout: default
...
         </div>
-        <div class=" mt-12 w-full max-w-2xl text-center flex gap-2 justify-between">
-            <div class="w-1/4 css-full">
-                <img src="./images/pictograms/security_red.png"/>
-                <h5 class="flex justify-center">SECURE</h5>
-                <label>Role-based access control at team and project levels.</label>
+        <div class="sm:mt-10 md:mt-16 w-full max-w-4xl text-center flex flex-wrap gap-2 justify-around">
+            <div class="w-full px-3 mt-4 sm:w-72 md:mt-0 css-full">
+                <img class="w-40 m-auto" src="./images/pictograms/community_red.png"/>
+                <h5 class="flex justify-center mb-2">SOCIALS & COMMS</h5>
```
Comment (joepavitt) [inline src/index.njk:34]:
```suggestion
                <h5 class="flex justify-center mb-2">SOCIAL & COMMS</h5>
```

**Review by sammachin — APPROVED** [review]

File: src/index.njk [hunk 2]
```diff
@@ -24,27 +24,40 @@ layout: default
...
+            </div>
+            <div class="w-full px-3 mt-4 sm:w-72 md:mt-0">
+                <img class="w-40 m-auto" src="./images/pictograms/candor_red.png"/>
+                <h5 class="flex justify-center mb-2">PROTOCOLS</h5>
+                <label class="font-light">WebSockets, HTTP, MQTT, AMQP, SNMP, RSS, FTP, etc.</label>
+            </div>
+            <div class="w-full px-3 mt-4 sm:w-72 lg:mt-10">
+                <img class="w-40 m-auto" src="./images/pictograms/cloud_red.png"/>
+                <h5 class="flex justify-center mb-2">CLOUD</h5>
+                <label class="font-light">AWS, Azure, S3, Google Cloup Platform, IBM Cloud, etc.</label>
```
Comment (kazuhitoyokoi) [inline src/index.njk:50]: @joepavitt There is a typo. Google Cloup Platform -> Google Cloud Platform

Comment (knolleary) [reply src/index.njk:50]: Fixed via #136

**Pull request merged**